src/note.ts

```diff
@@ -13,15 +13,254 @@
 import { Vex } from './vex';
 import { Flow } from './tables';
 import { Tickable } from './tickable';
+import { Stroke } from './strokes';
+import { Stave } from './stave';
+import { BoundingBox } from './boundingbox';
+import { Voice } from './voice';
+import { TickContext } from './tickcontext';
+import { ModifierContext } from './modifiercontext';
+import { Modifier } from './modifier';
+import { RenderContext } from './types/common';
+import { Fraction } from './fraction';
+import { Beam } from './beam';
+
+export const GLYPH_PROPS_VALID_TYPES: Record<string, Record<string, string>> = {
+  n: { name: 'note' },
+  r: { name: 'rest' },
+  h: { name: 'harmonic' },
+  m: { name: 'muted' },
+  s: { name: 'slash' },
+};
+
+export interface Coordinates {
+  x: number;
+  y: number;
+}
+
+export interface Font {
+  glyphs: { x_min: number; x_max: number; ha: number; o: string[] }[];
+  cssFontWeight: string;
+  ascender: number;
+  underlinePosition: number;
+  cssFontStyle: string;
+  boundingBox: { yMin: number; xMin: number; yMax: number; xMax: number };
+  resolution: number;
+  descender: number;
+  familyName: string;
+  lineHeight: number;
+  underlineThickness: number;
+  /**
+   * This property is missing in vexflow_font.js, but present in gonville_original.js and gonville_all.js.
+   */
+  original_font_information?: {
```

Comment:
This is a good place for `any`, I don't want to tie it too much to the TTF exported fields.

Reply:
Or an interface with just the fields that we use.

Reply:
Perhaps for this whole `Font` interface actually. It's noise in this file.

Reply:
do you want to move it to 'types/common'? The group from `@types/vexflow`made a great job figuring out the contents of the type. It is a pitty to lose it.

Reply:
Yeah, I understand, but I like to actively eliminate unnecessary code. We can always get it back if we need.

Reply:
`common` with `TODO`